packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -98,6 +98,64 @@ enum DragAnchor {
 /// widget always displays [child].
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=QzA4c4QHZCY}
+/// 
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// The follwing example has the [Draggable] widget along with the [DragTarget] in a row demonstrating what happens when you drag the element to the target.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   var acceptedData = 0;
+///   return Row(
+///     mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+///     children : [
+///       Draggable(
+///         // Data is the value this Draggable stores.
+///         data : 10,
+///         child : Container(
+///           height : 100.0,
+///           width : 100.0,
+///           color : Colors.lightGreenAccent,
+///           child : Center(
+///             child : Text("I am the Draggable")
+///           )
```

Comment:
Add trailing commas where needed, here and everywhere.
```suggestion
///           ),
```